integration-tests/maven/src/test/java/io/quarkus/maven/it/NativeImageIT.java

```diff
@@ -37,8 +34,13 @@ public void testJavaLibraryPathAtRuntime() throws Exception {
         final RunningInvoker running = new RunningInvoker(testDir, false);
 
         // trigger mvn package -Pnative -Dquarkus.ssl.native=true
-        final String[] mvnArgs = new String[] { "package", "-DskipTests", "-Pnative", "-Dquarkus.ssl.native=true" };
-        final MavenProcessInvocationResult result = running.execute(Arrays.asList(mvnArgs), Collections.emptyMap());
+        List<String> mavenArgs = Arrays.asList("package", "-DskipTests", "-Pnative", "-Dquarkus.ssl.native=true");
+        if (System.getenv("GRAALVM_HOME") == null || System.getenv("GRAALVM_HOME").isEmpty()) {
+            // If the GRAALVM_HOME env var isn't set, try to use docker to build the image.
+            // This is needed because CI doesn't have a GraalVM installation
+            mavenArgs.add("-Dquarkus.native.container-runtime=docker");
```

Comment:
Can't say I'm excited about this because that means we will require Docker to be installed for a standard JDK run? Or am I mistaken?

Reply:
Well unless I have completely missread this test, it can't succeed without GraalVM being installed, right?
So the change just allows us to fallback to docker if it's not.

Reply:
So essentially now it's safer than it was before, right?

Reply:
Fixed a different way